tests/killing-and-murder-err/program.txt

```diff
@@ -50,7 +47,7 @@ c: "y = &'L_*x mut *x" {
     introduce_subset('x, 'L_*x)
```

Comment:
We should find a way to make this not type check, cause this is now wrong. 

```suggestion
    introduce_subset('x, 'y)
```

(or can we make a test that fails without this new fact ?)